pkgs/by-name/cy/cyphernetes/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nodejs,
+  nix-update-script,
+  pnpm,
+  stdenvNoCC,
+  ...
+}:
+let
+  pname = "cyphernetes";
+  version = "0.14.0";
+  source = fetchFromGitHub {
+    owner = "AvitalTamir";
+    repo = "cyphernetes";
+    rev = "v${version}";
+    hash = "sha256-rxZWI+2eWMNTjphxS+lsxAoZUXD8wWF1pfe6hLBxtuM=";
+  };
+
+  cyphernetes-web-ui =
+    let
+      src = source + "/web";
+    in
+    stdenvNoCC.mkDerivation {
+      inherit version src;
+      pname = pname + "-web";
+      pnpmDeps = pnpm.fetchDeps {
+        inherit pname version src;
+        hash = "sha256-3uOBSVA4oyPDj9GLiBSPQonalaSYveJ4II01GTfaAUw=";
+      };
+      nativeBuildInputs = [
+        nodejs
+        pnpm.configHook
+      ];
+      buildPhase = ''
+        runHook preBuild
+        pnpm build
+        runHook postBuild
+      '';
+      installPhase = ''
+        runHook preInstall
+        mkdir -p $out
+        cp -R dist/* $out/
+        runHook postInstall
+      '';
+    };
+in
+buildGoModule {
+  inherit pname version;
+  src = source;
+  vendorHash = "sha256-DRzYgpHShZn+17R1Jj/arwAP5lyTpWelmwloUxT3n5Y=";
+  subPackages = [ "cmd/cyphernetes" ];
+  doCheck = false;
+  preBuild = ''
+    cp -r ${cyphernetes-web-ui} cmd/cyphernetes/web/
+  '';
+
+  meta = with lib; {
```

https://github.com/NixOS/nixpkgs/issues/208242

```suggestion
  meta = {
```